itou/templates/signup/prescriber_siren.html

```diff
@@ -0,0 +1,96 @@
+{% extends "layout/content_small.html" %}
+{% load bootstrap4 %}
+{% load static %}
+
+{% block title %}Prescripteur/Orienteur - Inscription{{ block.super }}{% endblock %}
+
+{% block content %}
+
+    <h1>
+        Inscription
+        <small class="text-muted">Prescripteur/Orienteur</small>
+    </h1>
+
+    <div class="alert alert-secondary" role="alert">
+        Retrouvez facilement votre numéro SIREN à partir du nom de votre organisation sur le site
+        <a href="https://sirene.fr/" rel="noopener" target="_blank">
+            sirene.fr
+            {% include "includes/icon.html" with icon="external-link" %}
+        </a>
+    </div>
+
+    <form method="post" action="" class="js-prevent-multiple-submit">
+
+        {% csrf_token %}
+
+        {% bootstrap_form form %}
+
+        {% buttons %}
+            {% if not form.cleaned_data %}
+            <a class="btn btn btn-outline-secondary" href="{{ prev_url }}">
+                Retour
+            </a>
+            <button type="submit" class="btn btn-primary">Continuer</button>
+            {% endif %}
+            {% if form.cleaned_data %}
+            <button type="submit" class="btn btn-primary">Rechercher</button>
+            {% endif %}
+        {% endbuttons %}
+
+    </form>
+
+    {% if not form.cleaned_data %}
+    <a href="{% url 'signup:prescriber_user' %}" >Je ne fais partie d'aucune organisation</a>
+    {% endif %}
+
+    {% if prescribers_with_members %}
+        <div class="mt-5">
+            <h3>Organisation(s) déjà inscrite(s)</h3>
+            <div class="alert alert-secondary" role="info">
+                <i>Par sécurité vous devez obtenir une invitation pour rejoindre une organisation déjà inscrite.</i>
+            </div>
+            <ul>
+                {% for prescriber in prescribers_with_members %}
+                    <li>
+                        <p>
+                            <b>{{ prescriber.siret|slice:":9" }} {{ prescriber.siret|slice:"9:" }}</b> - {{ prescriber.kind }}
+                            <br>
+                            {{ prescriber.display_name }}
+                            <br>
+                            {{ prescriber.address_line_1 }},
+                            {% if prescriber.address_line_2 %}{{ prescriber.address_line_2 }},{% endif %}
+                            {{ prescriber.post_code }} {{ prescriber.city }}
+                            <br>
+                            {% if prescriber.active_admin_members %}
+                                {% with prescriber.active_admin_members.first as admin %}
```

Comment:
Ici, l'accès à `.active_admin_members` peut déclencher un problème de requêtes N+1, puisque que tu es dans une boucle `for`.

Peut-être qu'il serait sage de vérifier dans la _debug toolbar_ qu'il n'y a pas un excès de requêtes dans certains cas et si c'est le cas, faire une jointure sur les membres dans la vue.

Reply:
Ah oui, en effet. Ce sera à optimiser ici aussi : https://github.com/betagouv/itou/blob/e66515c6290e0d7fa8691d7948678516703fe81f/itou/templates/signup/siae_select.html#L131